hasher-matcher-actioner/src/OpenMediaMatch/blueprints/matching.py

```diff
@@ -95,14 +97,19 @@ def raw_lookup():
      * Signal value (the hash)
      * Optional list of banks to restrict search to
     Output:
-     * List of matching content items
+     * List of matching with content_id and, if included, distance values
     """
     signal = require_request_param("signal")
     signal_type_name = require_request_param("signal_type")
-    return lookup_signal(signal, signal_type_name)
+    include_distance = bool(request.args.get("include_distance", False)) == True
+    lookup_signal_func = (
+        lookup_signal_with_distance if include_distance else lookup_signal
+    )
 
+    return lookup_signal_func(signal, signal_type_name)
```

Comment:
nit: Both of these functions return the same shape, it might be slightly easier to have this return the list and do


```
return {
  "matches": lookup_signal_func()
}
```